src/releasability/releasability_service.py

```diff
@@ -72,7 +71,7 @@ def start_releasability_checks(self, organization: str, repository: str, branch:
             organization=organization,
             project_name=repository,
             branch_name=branch,
-            version=standardized_version,
+            version=version,
```

Comment:
IMHO method `build_sns_request` signature should be updated so that it do not take `version=4.3.2.1232` and `build_number=1232` (this is redundant information `1232` and open the door to multiple sources of truth/conflicts what if version=4.3.2.1111 and given build_number=3232, the method will be lost)

As "version" **contains** the build number, it can be updated as follow:

```
    def _build_sns_request(
        self,
        correlation_id: str,
        organization: str,
        project_name: str,
        branch_name: str,
        revision: str,
        version: str,
        build_number: int, <--- this can be removed as it is included within version (it's a duplicate of information)
    ):

        build_number = VersionHelper.extractBuildNumber(version) // <--- here build_number is extracted from version

        sns_request = {
            'uuid': correlation_id,
            'responseToARN': self.RESULT_TOPIC_ARN,
            'repoSlug': f'{organization}/{project_name}',
            'version': version,
            'vcsRevision': revision,
            'artifactoryBuildNumber': build_number,
            'branchName': branch_name,
        }
        return sns_request
```

Available for more explanation if needed